### ISSUE TITLE
Latest activities : link of the profile picture and the name should be the user's profile page

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#3564]
### Is your feature request related to a problem? Please describe.

Go to https://community.humhub.com/dashboard
In the block "Latest activity", click on the profile picture or the name of a user that has just joined a space
You are redirected to the space

Eg :
![image](https://user-images.githubusercontent.com/23310825/58126523-6d362a80-7c13-11e9-9078-e18dfa15bfea.png)

### Describe the solution you'd like

The link should redirect to the user's profile or stream page, as it is the case in the rest of Humhub.

Eg :
![image](https://user-images.githubusercontent.com/23310825/58126606-9a82d880-7c13-11e9-8a45-1139cac9dddc.png)

**Comment by Felli** [comment]
The main purpose of the Latest Activity widget is to direct you to content, not a user's profile if someone wishes to get to someone's profile they can simply go through the post itself or view the content from the space, I completely understand what you're talking about, I just don't see a plus other than allowing users to be lazy and shortcutting. :thinking:

**Comment by funkycram** [comment]
Yes, but when somebody has just joined a space (first screenshot), the user has no post... So you are redirected to the main stream of the space, which is not so great !
